app/services/bootstrap_timeline_service.rb

```diff
@@ -28,7 +28,13 @@ def autofollow_bootstrap_timeline_accounts!
   def bootstrap_timeline_accounts
     return @bootstrap_timeline_accounts if defined?(@bootstrap_timeline_accounts)
 
-    @bootstrap_timeline_accounts = bootstrap_timeline_accounts_usernames.empty? ? admin_accounts : local_unlocked_accounts(bootstrap_timeline_accounts_usernames)
+    accounts = []
+
+    if Setting.enable_bootstrap_timeline_accounts || Setting.enable_bootstrap_timeline_accounts.nil?
```

Comment:
Please particularly pay attention to this line; I tried to make it enabled by default, is this good?

Reply:
Default values are set in `config/settings.yml`. Furthermore, I wouldn't edit the `bootstrap_timeline_accounts` method, but simply prevent `autofollow_bootstrap_timeline_accounts!` from running unless `Setting.enable_bootstrap_timeline_accounts` is true.